package.json

```diff
@@ -175,7 +176,7 @@
     "rimraf": "2.6.2",
     "sass-loader": "6.0.7",
     "serviceworker-webpack-plugin": "0.2.3",
-    "storybook-addon-vue-info": "0.1.8",
+    "storybook-addon-vue-info": "^0.2.0",
```

Comment:
can you remove the ^

Reply:
i culdn't start the app with older version..

Reply:
the version is fine. it's just to pin the exact version in package.json

Reply:
done!